install_config/aggregate_logging.adoc

```diff
@@ -468,81 +468,84 @@ endif::openshift-enterprise[]
 
-`RECOVER_AFTER_NODES` is the intended cluster size - 1.
-====
+- `*RECOVER_AFTER_NODES*` is the intended cluster size - 1.
+
+Edit the template of the cluster you are scaling up and update the three
+parameters as specified above.
 
-Edit the template of the cluster you will be scaling up and update the three parameters
-as specified above.
 ====
 ----
 $ oc edit template logging-es[-ops]-template
 ----
 ====
 
-In addition to updating the template, all the Deployment Configs for that cluster
-also need to have the three environment variable values above updated.  To edit
-each of the configs for the cluster in series you can use the following.
+In addition to updating the template, all of the deployment configurations for
+that cluster also need to have the three environment variable values above
+updated. To edit each of the configurations for the cluster in series, you use
+the following.
 
 ====
 ----
 $ oc get dc -l component=es[-ops] -o name | xargs -r oc edit
 ----
 ====
 
-Then to create an additional Deployment Configuration, run the following command against
-the Elasticsearch cluster you'd like to scale up for (`logging-es-template` or
-`logging-es-ops-template`).
+Then, to create an additional deployment configuration, run the following
+command against the Elasticsearch cluster you want to to scale up for
+(*logging-es-template* or *logging-es-ops-template*).
 
 ====
 ----
 $ oc new-app logging-es[-ops]-template
 ----
 ====
 
-These deployments will be named differently, but all will have the
-*logging-es* prefix. You should be aware of the cluster
-parameters (described in the deployer parameters) based on cluster size
-that may need corresponding adjustment in the template as well as
-existing deployments.
+These deployments will be named differently, but all will have the *logging-es*
+prefix. Be aware of the cluster parameters (described in the deployer
+parameters) based on cluster size that may need corresponding adjustment in the
+template, as well as existing deployments.
 
-After the intended number of Deployment Configs have been created, you can scale
-up your cluster, starting with Elasticsearch as described in steps 6-7
-link:../upgrading/manual_upgrades.html#manual-upgrading-efk-logging-stack[here].
+After the intended number of deployment configurations are created, scale up
+your cluster, starting with Elasticsearch as described in
+link:../install_config/upgrading/manual_upgrades.html#manual-upgrading-efk-logging-stack[Upgrading
+the EFK Logging Stack].
 
 [NOTE]
 ====
-With the above command, your Deployment Configuration will be created without
-a Persistent Volume. If you would like to create a replica with a Persistent
-Volume attached to it upon creation you can instead run the following command
-to create your DC with a PVC attached.
+With the above command, your deployment configuration will be created without a
+persistent volume. If you want to create a replica with a persistent volume
```

Comment:
Future tense, but also, which command are we talking about? The one from the link, or the new-app command? Maybe:

"The above command creates a deployment configuration with a persistent volume."